kaniko/code/main.go

```diff
@@ -115,17 +121,52 @@ func main() {
 	// Launch a timer to measure the time needed to parse/copy/extract
 	start := time.Now()
 
-	// Build the Dockerfile
-	logrus.Infof("Building the %s", b.DockerFileName)
-	err := b.BuildDockerFile()
-	if err != nil {
-		panic(err)
+	logrus.Info("Finding the Dockerfiles to build ...")
+	metadatafileNameToParse := os.Getenv("METADATA_FILE_NAME")
+	logrus.Infof("METADATA TOML FILE: %s", metadatafileNameToParse)
+	if metadatafileNameToParse == "" {
+		metadatafileNameToParse = defaultMetadataFileName
 	}
-	err = reapChildProcesses()
+
+	logrus.Info("Finding base image to provide ...")
+	baseimage := os.Getenv("CNB_BASE_IMAGE")
```

Comment:
Is it documented how to pass such ENV var part of the README ? Have you reviewed the Job.yaml definition to align it with your PR - https://github.com/redhat-buildpacks/poc/blob/main/helm/templates/job.yaml#L20-L49 ?